Cargo.toml

```diff
@@ -43,6 +43,8 @@ zeroize = { version = "1.4", features = ["zeroize_derive"], default-features = f
 thiserror = { version = "1.0", default-features = false }
 anyhow = { version = "1.0", default-features = false }
 
+dirs = "4.0"
```

Comment:
Keep default features off? They don't have any rn, but maybe in a future version they do

Reply:
That makes me sad 😞 .  Wouldn't it be great if we had a clean `Cargo.toml` file?